bespokefit_tutorial/bespoke_parameters_showcase.ipynb

```diff
@@ -0,0 +1,163 @@
+{
```

Comment:
<p>"any OpenFE protocol" - technically you're right, but we should be a bit careful just in case. How about "any OpenMM-based OpenFE Protocol"?</p><p> </p><p>"gathering results guide" - should all three words be part of the link (currently it's just the last two)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>I think it should work with all interchange supported exports as the proper torsion is a standard potential, but I'll switch to OpenMM only for now. </p>

Reply:
Yeah my main thing is more about the settings / API point might be different between Protocol types - I'm really just being overly cautious, so we don't have to make the switch if we don't feel like it.